modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -374,65 +374,69 @@ gcloud projects add-iam-policy-binding <service-project-id> \
+For example, if the GCP service account is `my-gcp-sa@my-project.iam.gserviceaccount.com`, then the account ID is `my-gcp-sa`.
++
+* Redpanda cluster service account
++
+.Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
-    --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
-gcloud iam service-accounts add-iam-policy-binding redpanda-gke@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-cluster-gcp-sa-account-id>]"
 ```
 ====
 
-. In the https://cloudv2.redpanda.com[Redpanda Cloud UI^], start to xref:./create-byoc-cluster-gcp.adoc[create a BYOC cluster] so you can get the generated Redpanda ID. This is necessary to create certain required resources.
-+
-IMPORTANT: On the *Network* page, select the *Customer-managed* network connection type, and enter the network, service account, and storage bucket information you created. However, before running the Validation `rpk` command, note the `redpanda-id` (for example, `cisld88gfi809ee1qjcg`). You finish the cluster deployment in a later step.  
-+
-. Use the `redpanda-id` to bind service accounts with the Workload Identity role.
-
-* Redpanda cluster service account
+* Redpanda Console service account
 +
 .Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-cluster@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/rp-<redpanda-id>]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<console-gcp-sa-account-id>]"
 ```
-==== 
-* Redpanda Console service account
+====
+
+* Redpanda Connectors service account
 +
 .Show command
 [%collapsible]
 ====
-```bash
-gcloud iam service-accounts add-iam-policy-binding redpanda-console@<service-project-id>.iam.gserviceaccount.com \
+```
+gcloud iam service-accounts add-iam-policy-binding <service-account-name>@<service-project-id>.iam.gserviceaccount.com \
     --role roles/iam.workloadIdentityUser \
-    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda/console-<redpanda-id>]"
+    --member "serviceAccount:<service-project-id>.svc.id.goog[redpanda-connectors/connectors-<connectors-gcp-sa-account-id>]"
```

Comment:
Why do we use a unique placeholder here? Is this service account ID intended to be different for Redpanda Console, connectors, and Redpanda clusters?

Reply:
@paulzhang97 can these 3 unique placeholders (in bindings for RP cluster SA, console SA, and connectors SA) all be replaced with `<gcp-sa-account-id>`?

Reply:
See https://github.com/redpanda-data/docs/pull/55#discussion_r1343085570